node-js-web-server/server080.js

```diff
@@ -0,0 +1,14 @@
+const http = require('http');
+
+http.createServer(function (request, response) {
+   console.log('request url: ', request.url);
+   
+   var filePath = './public' + request.url;
+}).listen(8080);
+console.log('Server is running at http://localhost:8080');
+
+// now everytime user hit an url you could see the file path.
```

Comment:
Now when someone visits `http://localhost:8080/about`, `filePath` will be `./public/about`.
But what about the home page, `http://localhos:8080/`?
There's no file named `/`, so we will need to account for that.
Write an `if` statement to check if `filePath` is equal to `./public/`.
If it is, set `filePath` equal to `./public/index.html`.